2021/23xxx/CVE-2021-23334.json

```diff
@@ -1,97 +1,99 @@
 {
```

Comment:
{} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'Arbitrary Code Execution'}]}]}